chapters/syntax.tex

```diff
@@ -260,12 +260,12 @@ \subsection{Equations}\label{equations1}
 
 \begin{lstlisting}[language=grammar]
 equation-section :
-   [ initial ] equation { equation ";" }
+   [ initial ] equation { some-equation ";" }
```

Comment:
We need to change both:
```suggestion
   [ initial ] some-equation { some-equation ";" }
```

Reply:
Hmm... No, that change can't be right. 

The first "equation" is the keyword (after optional "initial"). The second now named "some-equation" is zero or equation-grammar-constructs. I also checked https://modelica.org/documents/ModelicaSpec34.pdf and there the first is bolded as keyword, but not the second.

Reply:
My bad.  Resolving.